services/api/service.go

```diff
@@ -1888,8 +1886,14 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 	pf.RedisUpdate = uint64(nextTime.Sub(prevTime).Microseconds())
 	pf.Total = uint64(nextTime.Sub(receivedAt).Microseconds())
 
-	// All done
-	log.Info("received block from builder")
+	// All done, log with profiling information
+	log.WithFields(logrus.Fields{
```

Comment:
logs already are quite verbose, this seems more of a debug log.

Reply:
disagree, the timing information is critical, and it's only logging them once here, at the end of the request